src/index.html

```diff
@@ -1,13 +1,845 @@
+                  >
+                    Усі новини
+  
+                    <img 
+                      src="images/icons/btn-arrow.svg"
+                      alt="arrow"
+                      class="button--arrow-img"
+                    >
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="subscribe page__section">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="
+              grid__item
+              grid__item--tablet-1-4
+              grid__item--desktop-1-5"
+            >
+              <h3 class="subscribe__title">
+                Підпишіться на дайджест
+              </h3>
+
+              <p class="subscribe__desktop subscribe__description">
+                Першими дізнавайтесь про новини музею та розіграші, отримуйте запрошення на події та читайте статті від кураторів
+              </p>
+
+              <form
+                  action="#"
+                  method="get"
+                  id="form"
+                  class="form"
+              >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="e-mail"
+                  class="subscribe__fieled form__email"
+                  required
+                >
+              </form>
+
+              <button class="button subscribe__button">
+                Підписатись
+              </button>
+            </div>
+
+            <p class="
+              subscribe__hidden-element
+              grid__item
+              grid__item--desktop-7-11"
+            >
+              Першими дізнавайтесь про новини музею та розіграші, отримуйте запрошення на події та читайте статті від кураторів
+            </p>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="grid grid--desktop">
+          <div class="
+            grid__item
+            grid__item--tablet-1-2
+            grid__item--desktop-1-3"
+          >
+            <h5 class="footer__title">
+              Контакти
+            </h5>
+
+            <div class="footer__information">
+              <a
+                href="https://goo.gl/maps/JTiBL972muJwAfYd8"
+                class="page__link"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Київ,
+                <br>
+                вул. М. Грушевського, 6
+              </a>
+              <br>
+              <a href="tel:278-13-57" class="page__link">
+                тел. 278-13-57, 278-74-54
+              </a>
+              <br>
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://mail.google.com/mail/?view=cm&fs=1&tf=1&to=info@namu.kiev.ua"
+                class="page__link"
+              >
```

Comment:
Sorry if my comment confuse you. I mean that
```suggestion
              <a
                href="mailto:info@namu.kiev.ua"
                class="page__link"
              >
```

Reply:
What about fixing autofill, I can't do it
<img width="473" alt="image" src="https://user-images.githubusercontent.com/118539168/227461715-c3c0f9d9-3878-4e6a-9c76-9d9b19e28964.png">